project/maintainers.md

```diff
@@ -0,0 +1,237 @@
+<!-- vim: filetype=markdown colorcolumn=80
+-->
+
+# Maintainers handbook
+
+This document is a guide for maintainers across the spire/helm-chart effort to
+provide a consistent user interface for those going through the review process.
+
+## The Pull Request
+
+Pull requests are submitted through GitHub.  They are contributions to change
+the project.  There is no difference between code and non-code submissions, in
+procedure or policy.
+
+All maintainers should consider that pull requests are gifts.  The project
+survives due to the effort of frequent contributors and their generosity.  As
+such, to encourage future submissions, the default approach to handling a merge
+request should be gratitude, even if the request cannot be merged.
+
+## Roles
+
+There are two primary roles within the scope of a pull request.
+
+- Committer - one who offers code to be merged.
+- Maintainer - one who merges the code.
+
+To ensure that all code receives a proper review, these roles are mutually
+exclusive for each pull request.  Within the scope of a pull request, a
+Committer cannot be a Maintainer, nor can a Maintainer be a Committer.
+
+For abandoned pull requests still requiring work, or efforts transferred between
+people, a Maintainer may become a Committer, invalidating any of their prior
+review work.  
+
+### Committers
+
+Committers are people contributing changes to the repository.  The first
+committer is typically the one that opens the pull request.  Additional people
+can become committers in the same merge request if they change the pull request
+directly.
+
+Suggestions to a committer by a maintainer, such as commentary that with a
+change the merge request might be accepted, does not make a maintainer a
+committer, as the committer will choose include the change at their discretion. 
+
+### Maintainers
+
+Maintainers hold a dual role in the project.  They are ambassadors of the
+effort as well as the gatekeepers permitting changes to the repository.  As
+ambassadors, maintainers must present a fair and impartial demeanor when
+dealing with contributors.
+
+Failure to be fair or impartial reflects poorly on the released product, as
+guilt by association taints the product.  The process of reviewing a merge
+request often includes conflict.  Contributors can become defensive about work
+they've done while maintainers can become adamant in the changes they request.
+
+To prevent a breakdown in the review process, the project encourages all
+reviewers to adhere to a standard set of review best practices.  Reviewers
+should familiarize themselves with these practices and suggest updates to keep
+the practices relevant over time.
```

Comment:
```suggestion
To prevent a breakdown in the review process, the project encourages all
reviewers to adhere to a standard set of review best practices. Reviewers
should familiarize themselves with these practices and suggest updates to keep
the practices relevant over time.
```